domino/http_request_manager.py

```diff
@@ -33,6 +36,9 @@ def _raise_for_status(self, response):
         try:
             response.raise_for_status()
         except requests.exceptions.HTTPError as e:
+            if e.response.status_code == HTTPStatus.CONFLICT:
+                parse_result = urlparse(e.response.url)
+                self._logger.error(f" An error has occurred. Try to relogin at {parse_result.scheme}://{parse_result.netloc}")
```

Comment:
I don't think it's an error condition. I would treat is as a normal outcome and say something like "In order to start your _execution_ you need to login to the Domino UI, please do so at <url>"

But in this case you'd have to throw a _very_ specific exception here, and catch it in  starting job and staring workspace.